src/components/home/example.tsx

```diff
@@ -1,8 +1,15 @@
 const example = () => {
   return (
-    <div className="flex items-center justify-center text-[10vw] text-cvdsa-red-200">
-      example
-    </div>
+    <>
+      <div className="flex items-center justify-center text-[10vw] text-cvdsa-red-200">
+        example
+      </div>
+      <div className="flex items-center justify-center">
+        <button className="mt-56 rounded-full border px-6 py-2 text-3xl">
```

Comment:
```suggestion
        <button className="rounded-full border px-6 py-2 text-3xl">
```
add a shadow below the button -- reference the figma, I think the shadow is only below the button and not any other angle